src/Apps/ArtQuiz/ArtQuizWelcome.tsx

```diff
@@ -26,55 +25,54 @@ export const ArtQuizWelcome: FC<ArtQuizWelcomeProps> = ({ onStartQuiz }) => {
     >
-      <Box height="100%" padding={0}>
-        <SplitLayout
-          hideLogo
-          left={
-            <Flex height="100%" alignItems="center" justifyContent="center">
-              <ArtsyMarkBlackIcon height="65px" width="65px" fill="white100" />
-            </Flex>
-          }
-          leftProps={{ display: ["none", "block"] }}
-          right={
-            <Flex flexDirection="column" justifyContent="center" p={[2, 4]}>
-              <Box width="100%" my={6}>
-                <ArtsyLogoIcon mb={2} />
-                <Text variant={["xl", "xxl"]}>{t("artQuizPage.title")}</Text>
+      <SplitLayout
+        hideLogo
+        left={
+          <Flex height="100%" alignItems="center" justifyContent="center">
+            <ArtsyMarkBlackIcon height={65} width={65} fill="white100" />
+          </Flex>
+        }
+        leftProps={{ display: ["none", "block"] }}
+        right={
+          <Flex flexDirection="column" justifyContent="center" p={[2, 4]}>
+            <ArtsyLogoIcon />
 
-                <Spacer my={6} />
+            <Spacer y={2} />
 
-                <Text variant={["md", "lg"]}>
-                  {t("artQuizPage.welcomeScreen.subtitle1")}
-                </Text>
+            <Text variant={["xl", "xxl"]}>{t("artQuizPage.title")}</Text>
 
-                <Spacer my={2} />
+            <Spacer y={6} />
 
-                <Text variant={["md", "lg-display"]}>
-                  {t("artQuizPage.welcomeScreen.subtitle2")}
-                </Text>
-              </Box>
+            <Text variant={["md", "lg"]}>
+              {t("artQuizPage.welcomeScreen.subtitle1")}
+            </Text>
 
-              <Spacer my={6} />
+            <Spacer y={2} />
 
-              <Box width="100%" my={6}>
-                <Button width="100%" onClick={onStartQuiz}>
-                  {t("artQuizPage.welcomeScreen.getStartedButton")}
-                </Button>
-                <Button
-                  // @ts-ignore
-                  as={RouterLink}
-                  variant="tertiary"
-                  mt={1}
-                  width="100%"
-                  to="/"
-                >
-                  {t("artQuizPage.welcomeScreen.skipButton")}
-                </Button>
-              </Box>
-            </Flex>
-          }
-        />
-      </Box>
+            <Text variant={["md", "lg-display"]}>
+              {t("artQuizPage.welcomeScreen.subtitle2")}
+            </Text>
+
+            <Spacer y={6} />
+
+            <Button width="100%" onClick={onStartQuiz}>
+              {t("artQuizPage.welcomeScreen.getStartedButton")}
+            </Button>
+
+            <Spacer y={1} />
+
+            <Button
+              // @ts-ignore
+              as={RouterLink}
+              variant="tertiary"
+              width="100%"
+              to="/"
+            >
+              {t("artQuizPage.welcomeScreen.skipButton")}
+            </Button>
+          </Flex>
+        }
+      />
```

Comment:
OK I should have left this one alone to minimize the conflict; lemme roll that off